docs/howtos/layers/shapes.md

```diff
@@ -384,38 +527,24 @@ to the edge and face colors, these property is different from the
 `layer.current_edge_width` property that will determine the edge width of the
 next shape to be added or any currently selected shapes.
 
-To change the edge with property from the GUI you must first select the shape
-whose properties you want to change, otherwise you will just be initializing the
-property for the next shape you add.
-
-## Layer ordering
-
-You can adjust the ordering of shapes in the layer by selecting shapes and then
-clicking the move to front or move to back buttons. You can get the ordering of
-all the shapes using the `layer.z_indices` property. You can also set the
-initial ordering of shapes by passing a list to the `layer.z_index` property.
-
-## Copying and pasting shapes
+### Layer ordering
 
-It is possible to copy and paste any selected shapes using the `ctrl-C` and
-`ctrl-V` keybindings respectively. If you have a multidimensional shapes layer
-you can copy shapes from one slice to another by pasting them into the new
-slice. The coordinates of the shapes in the visible dimensions will be in the
-same place on the new slice as in the old slice, but the rest of the coordinates
-will be updated with the new slice values.
+You can get the ordering of all the shapes using the `layer.z_indices` property.
+You can also set the initial ordering of shapes by passing a list to the
+`layer.z_index` property.
 
-## Shapes layer opacity
+### `shapes` layer opacity
 
-The {ref}`opacity value <layer_opacity>` applies to all shapes. You can 
+The {ref}`opacity value <layer_opacity>` applies to all shapes. You can
 initialize the shape opacities using the `opacity` keyword argument which
 accepts a single opacity value that will be applied globally. You can then
 access the opacity using the `layer.opacity` property. In order to adjust the
 opacity of individual shapes you need to adjust the alpha value in the
 `layer.edge_color` and `layer.face_color` properties.
 
-## Putting it all together
+### Putting it all together
 
-Here you can see an example of adding, selecting, and editing shapes and change
-their properties:
+Here you can see an example of adding, selecting, and editing shapes and
+changing their properties:
 
 ![image: editing shapes](../../images/editing_shapes.webm)
```

Comment:
This video is also visibly dated, because colors are set using a color picker now.